docs/shared/packages/powerpack-azure-cache/powerpack-azure-cache-plugin.md

```diff
@@ -81,3 +81,29 @@ Finally, you need to configure your Nx cache in the `nx.json` file. The `contain
 | --------------- | -------------------------------- |
 | **container**   | The name of the container to use |
 | **accountName** | The name of blob storage account |
+
+# Cache Modes
+
+By default, the remote cache will try to write and read from the remote cache while running locally. This means that permissions must be set for users who are expected to access the remote cache.
```

Comment:
```suggestion
By default, Nx will try to write and read from the remote cache while running locally. This means that permissions must be set for users who are expected to access the remote cache.
```

Reply:
Not sure what the subject of this sentence should be, but I don't the remote cache is reading from itself.  This comment applies to all the cache docs.

Reply:
Maybe the subject should be "the Powerpack Azure Cache Plugin"